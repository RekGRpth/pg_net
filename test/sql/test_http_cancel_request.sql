begin;

    with req as (
        select net.http_get('https://supabase.io') as id
    )
    select
        net.http_cancel_request(id) is not null
    from
        req;

rollback;
