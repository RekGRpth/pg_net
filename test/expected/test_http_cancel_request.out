begin;
    with req as (
        select net.http_get('net://supabase.io') as id
    )
    select
        net.http_cancel_request(id) is not null
    from
        req;
 ?column? 
----------
 t
(1 row)

rollback;
